### PR TITLE
Document how to get past bad jobs

### PR DESCRIPTION
**Pull request by carolyncole**

### CODE REVIEW

File: docs/bad_dump [hunk 1]
```diff
@@ -0,0 +1,8 @@
+1. sssh worker box
```
Comment (sandbergja) [inline docs/bad_dump:1]: sssh should be ssh

**Review by sandbergja — COMMENTED** [review]
Thank you for these, Carolyn!  Should this have the .md extension so it displays nicer in Github?

**Comment by carolyncole** [comment]
Thanks @sandbergja for the review! @christinach I fixed up a couple changes @sandbergja asked about.  Feel free to rearrange when adding in the actual commands.

**Review by christinach — DISMISSED** [review]

File: docs/bad_dump.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+1. `ssh deploy@bibdata-alma-worker1`
+1. `cd /opt/marc_liberation/current`
+1. `RAILS_ENV=production bundle exec rails c`
+1. Find the IndexManager instance that uses the solr production collection. Currenlty is:`catalog-alma-production`
+`index_manager_current = IndexManager.all.first`
+1. `index_manager_current.solr_collection` => `http://lib-solr8-prod.princeton.edu:8983/solr/catalog-alma-production`
```
Comment (carolyncole) [inline docs/bad_dump.md:6]: @christinach why is this a step?  What is it doing?

**Comment by christinach** [comment]
@sandbergja can you re-review?

File: docs/bad_dump.md (hunk 2 above)
Comment (christinach) [reply docs/bad_dump.md:6]: its not a step. You make sure that you are checking the correct index manager instance by checking the solr_collection. I'm sorry I added it as a step.

Comment (christinach) [reply docs/bad_dump.md:6]: there are two instances one has as a solr_collection: catalog-alma-production-rebuild but it is not in production.

Comment (christinach) [reply docs/bad_dump.md:6]: I will update it

File: docs/bad_dump.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+1. `ssh deploy@bibdata-alma-worker1`
+1. `cd /opt/marc_liberation/current`
+1. `RAILS_ENV=production bundle exec rails c`
+1. Find the IndexManager instance that uses the solr production collection. Currenlty is:`catalog-alma-production`
+`index_manager_current = IndexManager.all.first`
```
Comment (christinach) [inline docs/bad_dump.md:5]: We should update this. We need to find the current index manager which is in production. It might not be the first.

**Review by sandbergja — APPROVED** [review]

**Pull request merged**